Framework/Algorithms/src/CopyDataRange.cpp

```diff
@@ -99,31 +99,22 @@ void CopyDataRange::init() {
 std::map<std::string, std::string> CopyDataRange::validateInputs() {
   std::map<std::string, std::string> errors;
 
-  auto const inputWorkspaceName = getPropertyValue("InputWorkspace");
   MatrixWorkspace_sptr inputWorkspace = getProperty("InputWorkspace");
-  auto const destWorkspaceName = getPropertyValue("DestWorkspace");
   MatrixWorkspace_sptr destWorkspace = getProperty("DestWorkspace");
-  auto const outputWorkspaceName = getPropertyValue("OutputWorkspace");
 
   int const specMinIndex = getProperty("StartWorkspaceIndex");
   int const specMaxIndex = getProperty("EndWorkspaceIndex");
-  int const xMinIndex = getProperty("XMinIndex");
-  int const xMaxIndex = getProperty("XMaxIndex");
+  int const xMinIndex = getProperty("XMin");
```

Comment:
My apologies I don't think I explained myself too well here. I meant have `XMin` and `XMax` be the X values and not the indices as `CropWorkspace` does. It feels a little inconsistent to have one take indices and one take values. Would this be possible?

Reply:
Ahh I understand now 👍